### PR TITLE
Adding super calls to JIT test case setUp and tearDown

### PR DESCRIPTION
**Pull request by janeyx99**
This issue was surfaced when adding this issue: https://github.com/pytorch/pytorch/issues/61655 did not manage to skip the appropriate test case. 

I then investigated and realized it was because the setUp code that does the test disabling is not called because another defined setUp overrode the parent class' setUp.

I am not sure if that was intentional--if so we would have to adopt the child class' code to call the check_if_enable function in common_utils.

### CODE REVIEW

File: test/test_cpp_extensions_jit.py [hunk 1]
```diff
@@ -47,12 +47,14 @@ class TestCppExtensionJIT(common.TestCase):
     def setUp(self):
         # cpp extensions use relative paths. Those paths are relative to
         # this file, so we'll change the working directory temporarily
+        super().setUp()
```
Comment (samestep) [inline test/test_cpp_extensions_jit.py:50]: might it be clearer to put the comments right next to what they're referring to?
```suggestion
        super().setUp()
        # cpp extensions use relative paths. Those paths are relative to
        # this file, so we'll change the working directory temporarily
```

File: test/test_cpp_extensions_jit.py [hunk 2]
```diff
@@ -47,12 +47,14 @@ class TestCppExtensionJIT(common.TestCase):
...
         self.old_working_dir = os.getcwd()
         os.chdir(os.path.dirname(os.path.abspath(__file__)))
 
     def tearDown(self):
         # return the working directory (see setUp)
         os.chdir(self.old_working_dir)
+        super().tearDown()
```
Comment (samestep) [inline test/test_cpp_extensions_jit.py:57]: similar idea here
```suggestion
        super().tearDown()
        # return the working directory (see setUp)
        os.chdir(self.old_working_dir)
```

**Review by samestep — APPROVED** [review]
lgtm :+1: very strange

**Review by zou3519 — APPROVED** [review]
> I then investigated and realized it was because the setUp code that does the test disabling is not called because another defined setUp overrode the parent class' setUp.

Not intentional, thanks for catching this

**Pull request closed without merging**